crypto/keys/handlers.go

```diff
@@ -27,8 +28,11 @@ import (
 // For example, the protobuf message may contain a key or identify where a key can be found.
 type ProtoHandler func(context.Context, proto.Message) (crypto.Signer, error)
 
-// handlers convert a protobuf message into a crypto.Signer.
-var handlers = make(map[string]ProtoHandler)
+var (
+	// handlers convert a protobuf message into a crypto.Signer.
+	handlers   = make(map[string]ProtoHandler)
+	handlersMu sync.RWMutex
```

Comment:
nit: might as well call this handlersMutex?

Reply:
`Mu` is a pretty standard prefix for Mutex in Go code